spec/integration/test.js

```diff
@@ -109,6 +109,22 @@ describer('notification api with a live service', function () {
       })
     });
 
+    it('send email notification with CSV document upload', () => {
+      var postEmailNotificationResponseJson = require('./schemas/v2/POST_notification_email_response.json'),
+        options = {personalisation: { name: 'Foo', documents:
+          notifyClient.prepareUpload(Buffer.from("a,b"), true)
+        }, reference: clientRef};
+
+      return notifyClient.sendEmail(emailTemplateId, email, options).then((response) => {
+        response.statusCode.should.equal(201);
+        expect(response.body).to.be.jsonSchema(postEmailNotificationResponseJson);
+        response.body.content.body.should.equal('Hello Foo\r\n\r\nFunctional test help make our world a better place');
```

Comment:
So I've duplicated this test from the integration test of doc download above. However, I wonder if it's actually working given it appears the template doesn't contain a second placeholder for documents so this is probably us just passing an additional placeholder that the API ignores.

It should still be testing that the request can be made and it doesn't malform something and get a 500 response but I think this test probably isn't testing as much as we would like.

Saying that, I'm looking to get this in quickly as it will give the functionality to a COVID team so hopefully it's acceptable as is and I can come back and really sort out these tests properly later.

Reply:
I'd say to drop a chore in pivotal to sort it out later. Sounds like it could involve adding a template, environment variable and credential before dealing with any issues caused by updating these tests so probably best to keep it out of these changes